modules/Core/pages/user/sessions.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ *  Made by Aberdeener
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.1.0
+ *
+ *  License: MIT
+ *
+ *  User sessions page
+ */
+
+// Must be logged in
+if (!$user->isLoggedIn()) {
+    Redirect::to(URL::build('/'));
+}
+
+// Always define page name for navbar
+const PAGE = 'cc_sessions';
+$page_title = $language->get('user', 'user_cp');
+require_once(ROOT_PATH . '/core/templates/frontend_init.php');
+
+$timeago = new TimeAgo(TIMEZONE);
+
+if (Input::exists()) {
+    if (Token::check()) {
+        if (Input::get('action') === 'logout_other_sessions') {
+            $user->logoutAllOtherSessions();
+            Session::flash('user_sessions_success', $language->get('user', 'sessions_logged_out'));
+        }
+    } else {
+        // Invalid token
+        Session::flash('user_sessions_error', $language->get('general', 'invalid_token'));
+    }
+}
+
+if (Session::exists('user_sessions_success')) {
+    $smarty->assign([
+        'SUCCESS' => $language->get('general', 'success'),
+        'SUCCESS_MESSAGE' => Session::flash('user_sessions_success'),
+    ]);
+}
+
+if (Session::exists('user_sessions_error')) {
+    $smarty->assign([
+        'ERROR' => $language->get('general', 'error'),
+        'ERROR_MESSAGE' => Session::flash('user_sessions_error'),
+    ]);
+}
+
+$sessions = $user->getActiveSessions();
+$sessions_list = [];
+// TODO: Should we display all sessions, or just active ones? Over time, the list could get very long if we display all sessions.
+// Not really any reason to show inactive ones, since they can't action on them.
```

Comment:
Showing inactive ones can help us find if the account has been compromised or not, Hiding sessions could miss unwanted logins